### DISCUSSION TITLE
Laragon crashes a lot, is it just me?

### DISCUSSION THREAD

**Discussion by mayankguptadotcom** [leokhoa/laragon#414] [General]
I've recently come across Laragon, and right away I noticed that it crashes a lot. Is it something with my setup or are other users facing this issue too?

**Comment by dustinatx** [comment]
It has only crashed for me when attempting to use ngrok. Other than that, I've never had a crash. And I've been using it for a long time.

**Comment by JenuelDev** [comment]
its fine with me after upgrading to latest version.

**Comment by Pen-y-Fan** [comment]
There are some known problems with Laragon, especially switching to PHP 8 while Apache is running. Take a look at my blog series, there are probably some tips which will help:

- [How to set up Laragon on a new Windows computer (part 1) - Install Laragon](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/)

Once Laragon is set up and the known problems worked around it is very stable.
